docs/references/express/migrate-from-node-to-express.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Clerk Express.js SDK
+description: Learn how to migrate from Node.js to Express.js.
+---
+
+# Clerk Express.js SDK
+
+## Migrating from Clerk Node SDK to Clerk Express
+
+<Steps>
+
+### Uninstall @clerk/clerk-sdk-node and install @clerk/express
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm uninstall @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+yarn remove @clerk/clerk-sdk-node
+```
+
+```sh filename="terminal"
+pnpm remove @clerk/clerk-sdk-node
+```
+
+</CodeBlockTabs>
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+### Require Authentication
+
+**Using Node SDK**
+
+To enforce strict authentication, use `ClerkExpressRequireAuth()`. This middleware will raise an error when an unauthenticated request is made.
+You can find an example [here](/docs/backend-requests/handling/nodejs#clerk-express-require-auth).
+
+**Using Express SDK**
+
+- **Centralized Middleware:**
+Use `clerkMiddleware` to authenticate all incoming requests without blocking them. This middleware will handle the handshake mechanism.
+- **Require Authentication in Specific Routes or Handlers:**
+Use `requireAuth()` to enforce authentication on specific routes. This middleware returns an HTTP 401 response for unauthenticated request.
```

Comment:
🔧  Use `requireAuth`. You don't need to call the method, it's a request handler.